src/pages/_document.tsx

```diff
@@ -16,9 +15,7 @@ export default class MyDocument extends Document {
 	render() {
 		return (
 			<Html lang="en">
-				<Head>
-					<HashiHead />
-				</Head>
+				<Head></Head>
```

Comment:
`next/document` head is not the same as `next/head`, [`next/document` is used for properties that does not change across all pages.](https://nextjs.org/docs/pages/building-your-application/routing/custom-document#caveats)

[We are already rendering HashiHead in our _app file.](https://github.com/hashicorp/dev-portal/blob/main/src/pages/_app.tsx#L103) So this one is unnecessary.